From ecf73e7a2bcabb3e6345f281f9f6607f1276a7ad Mon Sep 17 00:00:00 2001
From: Yichao Yu <yyc1992@gmail.com>
Date: Sun, 16 Jan 2022 22:00:50 -0500
Subject: [PATCH 1/2] Do not load host file system

This cause loading failure if nodefs isn't linked in
and it's also a security vulnerability to require exposing the host fs.

Nothing in the code actually use this and this is completely useless.
---
 kernel/driver.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/driver.cc b/kernel/driver.cc
index 65f090993..4d32b13d5 100644
--- a/kernel/driver.cc
+++ b/kernel/driver.cc
@@ -70,7 +70,7 @@ extern "C" const char *prompt();
 int main(int argc, char **argv)
 {
 	EM_ASM(
-		if (ENVIRONMENT_IS_NODE)
+		if (0 && ENVIRONMENT_IS_NODE)
 		{
 			FS.mkdir('/hostcwd');
 			FS.mount(NODEFS, { root: '.' }, '/hostcwd');
-- 
2.48.1

